internal/controller/model_controller.go

```diff
@@ -35,14 +36,15 @@ const (
 	TrainingDatasetNotReady   = "TrainingDatasetNotReady"
 
 	modelTrainerServiceAccountName = "model-trainer"
-	modelBuilderServiceAccountName = "model-builder"
 )
 
 // ModelReconciler reconciles a Model object.
 type ModelReconciler struct {
 	client.Client
 	Scheme *runtime.Scheme
 
+	*builder.ContainerReconciler
```

Comment:
This covers both use cases of trainer and loader, right?

Reply:
Yep, it just builds images with kaniko